app/controllers/carts_controller.rb

```diff
@@ -25,6 +25,11 @@ def remove_item
     redirect_back(fallback_location: (request.referer || restaurant_path(params[:restaurant_id])))
   end
 
+  def clear_cart
+    @cart.clear
+    redirect_to root_path
```

Comment:
Do we want to go to root path or to the restaurant we were on?

Reply:
I guess you want the restaurant, or else you wouldn't comment? :)

Reply:
I prefer back to the restaurant. but its a matter of preference.

Reply:
I can go with that.